lib/hmac_auth/rack.rb

```diff
@@ -0,0 +1,38 @@
+require 'hmac_auth/x_headers'
```

Comment:
Yeah, that bug is 4 years old. Autoload was an issue in the pre-2.0 days because of thread safety concerns, but my understanding is it's been fixed in all versions beyond 1.9. A bunch of libraries (like Rails) use it successfully.